.github/workflows/ci-cd.yml

```
@@ -0,0 +1,292 @@
+      - name: Merge test results
+        run: |
+          mkdir -p merged-results/allure-results
+          find artifacts/ -name "*.json" -path "*/allure-results/*" -exec cp {} merged-results/allure-results/ \;
+
+      - name: Generate Allure report
+        run: |
+          npm run report:allure -- --clean merged-results/allure-results --output allure-report
+        continue-on-error: true
+
+      - name: Deploy reports to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)